test/webhook-proxy.test.ts

```diff
@@ -1,13 +1,19 @@
-const express = require('express')
+import express from 'express'
+import { Server } from 'http'
+import nock from 'nock'
+import { logger } from '../src/logger'
+import { createWebhookProxy } from '../src/webhook-proxy'
+
+// tslint:disable-next-line:no-var-requires
 const sse = require('connect-sse')()
-const nock = require('nock')
-const {createWebhookProxy} = require('../src/webhook-proxy')
-const {logger} = require('../src/logger')
 
 const targetPort = 999999
 
 describe('webhook-proxy', () => {
-  let app, server, proxy, url, emit
```

Comment:
Instead of declaring `url` at this scope, we can just declare `const url` further down in a the couple tests that use it and narrow the scope of the variable a little bit more. 👍